courses/additional_modules/multiple_inheritance.rst

```diff
@@ -0,0 +1,175 @@
+
+**********************
+Multiple Inheritance
+**********************
+
+==============
+Introduction
+==============
+
+------------------------------------------
+Multiple Inheritance Is Forbidden In Ada
+------------------------------------------
+
+* There are potential conflicts with multiple inheritance
+* Some languages allow it:ambiguities have to be resolved when entities are referenced
+* Ada forbids it to improve integration
+    
+.. code:: Ada
+    
+    type Graphic is tagged record
+       X, Y : Float;
+    end record;
+    function Get_X (V : Graphic) return Float;
+       
+    type Shape is tagged record
+       X, Y : Float;
+    end record;
+    function Get_X (V : Shape) return Float;
+       
+    type Displayable_Shape is new Shape and Graphic with ...
+
+----------------------------------
+Multiple Inheritance - Safe Case
```

Comment:
```suggestion
Multiple Inheritance |mdash| Safe Case
```
**Note**: `|mdash|` is only available when you include `isopub.txt` (using `.. include:: <isopub.txt>`).

Reply:
Pandoc does not give us "mdash" for free, so I'm going to leave this as-is for now